apps/src/p5lab/AnimationTab/AnimationList.jsx

```diff
@@ -20,23 +20,39 @@ class AnimationList extends React.Component {
     onNewItemClick: PropTypes.func.isRequired,
     spriteLab: PropTypes.bool.isRequired,
     hideBackgrounds: PropTypes.bool.isRequired,
+    hideCostumes: PropTypes.bool.isRequired,
     labType: PropTypes.string.isRequired
   };
 
   render() {
+    let buttonLabelNewAnimation;
```

Comment:
naming nit: `newAnimationLabel` might be enough to convey what this variable does